app/models/chat.py

```diff
@@ -0,0 +1,88 @@
+from sqlalchemy import String, Text, Enum, ForeignKey
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from uuid import UUID
+from .base import Base
+from .user import User
+from enum import Enum as PyEnum
+from category import CategoryEnum
```

Comment:
위에 `base`나 `user`처럼 import하는 형식이 똑같아야 할 것 같아요!

```suggestion
from .category import CategoryEnum
```